src/metabase/lib/convert.cljc

```diff
@@ -154,23 +154,28 @@
         (dissoc :source-table))
     stage))
 
+(defn do-with-aggregation-list
```

Comment:
I moved the macro implementation into its own function to make errors here easier to debug, otherwise the stacktrace will give every line in here as being at the line `with-aggregation-list` is used in